documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md

```diff
@@ -101,6 +101,6 @@ From this, we can conclude that we have successfully minted 1000 tokens with 40
 
 ## What next?
 
-Now that you have minted your tokens, you need to make sure that it stays collateralized as the value of the underlying moves (e.g. the ETH to BTC ratio). You can do this with the Sponsor [CLI tool](./using_the_uma_sponsor_cli_tool.md) that helps you manage your position. In the future, we might provide a dapp frontend for your convenience. Let us know on our Discord if you want to see this.
+Now that you have minted your tokens, you need to make sure that it stays collateralized as the value of the underlying moves (e.g. the ETH to BTC ratio, or the price of COMP). You can do this with the Sponsor [CLI tool](./using_the_uma_sponsor_cli_tool.md) that helps you manage your position. In the future, we might provide a dapp frontend for your convenience. Let us know on our Discord if you want to see this.
```

Comment:
nit: you might want to specify the price of comp "in dollars" unless you think that's obvious.

Reply:
If you want to change this, it can be done in a later PR so you don't have to wait on ci.

Reply:
I'll add it and merge, thanks for looking!

Reply:
d'oh, too late 😢